### PR TITLE
Change the Rally repository URL and default ref

### PR DESCRIPTION
**Pull request by Dexterp37**
We'll be cutting new releases from the `release` branch, so I changed `default-ref` too. Is that what is that property for?

### CODE REVIEW

**Review by escapewindow — CHANGES_REQUESTED** [review]
I think we have to update https://github.com/mozilla-extensions/xpi-manifest/blob/master/manifests/rally-core-addon.yml to point to the `release` branch as well.

Will we *only* release off the release branch, or another branch as well?

**Comment by Dexterp37** [comment]
> 
> 
> I think we have to update https://github.com/mozilla-extensions/xpi-manifest/blob/master/manifests/rally-core-addon.yml to point to the `release` branch as well.

Thanks!

> Will we _only_ release off the release branch, or another branch as well?

I don't expect to release from branches other than 'release' anytime soon (but it might happen longer term if we want, for example, to release a point release for an older version that's unsupported by newer browser versions. very unlikely!)

File: manifests/rally-core-addon.yml [hunk 1]
```diff
@@ -2,6 +2,7 @@ description: Rally Core Add-on
 repo-prefix: rallycoreaddon
 active: true
 private-repo: false
+branch: release
```
Comment (escapewindow) [inline manifests/rally-core-addon.yml:5]: I think this will work. We may want to test on ship-it today, so we know.

**Review by escapewindow — APPROVED** [review]

File: manifests/rally-core-addon.yml (hunk 1 above)
Comment (Dexterp37) [reply manifests/rally-core-addon.yml:5]: Thanks. I'll cut a new release right now.

**Pull request merged**